docs/getting_started/python/first_program_in_python/index.md

```diff
@@ -218,7 +219,7 @@ First, the `withdraw()` Activity takes the details about the transfer and calls
         reference_id = f"{data.reference_id}-withdrawal"
         try:
             confirmation = await asyncio.to_thread(
-                self.bank.withdraw, data.source_account, data.amount, 
+                self.bank.withdraw, data.source_account, data.amount, reference_id
```

Comment:
Did this maybe slip into the commit?

Reply:
It almost certainly did. I'll remove it from the commit.